cmd/create.go

```diff
@@ -375,6 +379,14 @@ func createHost(provider, name, region, zone, projectID, userData, inletsPort st
 				"pro":         fmt.Sprint(pro),
 			},
 		}, nil
+	} else if provider == "hetzner" {
+		return &provision.BasicHost{
+			Name:       name,
+			Region:     region,
+			Plan:       "cx11",
+			OS:         "ubuntu-20.04",
```

Comment:
We use 16.04 elsewhere which is faster and smaller. Is it an option in Hetzner ?

Also it would helpful if there is a link here to where these information can be found.

Reply:
Absolutely, fixed! :)